.codeclimate.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
It's an old format, would be better to use [new](https://docs.codeclimate.com/docs/advanced-configuration)